docs/book/src/quick-start.md

```diff
@@ -12,9 +12,15 @@ This Quick Start guide will cover:
 - [go](https://golang.org/dl/) version v1.15+.
 - [docker](https://docs.docker.com/install/) version 17.03+.
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) version v1.11.3+.
-- [kustomize](https://sigs.k8s.io/kustomize/docs/INSTALL.md) v3.1.0+
 - Access to a Kubernetes v1.11.3+ cluster.
 
+<aside class="note">
+<h1>Versions</h1>
+
+Kubebuilder will create a project for you where the officially versions supported are defined on its Makefile and go.mod file. For example,  see that it will install a specific version for Kustomize which is a prerequisite for the tool.  
```

Comment:
Can you make a list of what these binaries are?

Reply:
would not it be too verbose and may outdated easily? How do you think that it should be? Could you please give an example?

Reply:
We should be listing project dependencies, and updating that list when we make changes. We don't add/remove tools often so I don't see this list getting outdated quickly.
```suggestion
Projects created by kubebuilder contain a Makefile that will install tools at versions defined at creation time. Those tools are:
- [kustomize](https://kubernetes-sigs.github.io/kustomize/)
- [controller-gen](https://github.com/kubernetes-sigs/controller-tools)
```

Reply:
Done: 

<img width="781" alt="Screen Shot 2020-10-22 at 16 08 28" src="https://user-images.githubusercontent.com/7708031/96918548-5fd53d80-14a2-11eb-9a7d-b78ee64172dc.png">

Thanks for your feedback.